app/pods/search/agenda-items/template.hbs

```diff
@@ -97,6 +97,22 @@
           <WebComponents::VlAlert @message={{t "no-results-found-modify-term"}}/>
         {{/if}}
       {{/if}}
+      <div class="vlc-toolbar__item vl-u-display-flex vl-u-flex-align-center vlc-search__sizeoptions">
+        <label class="vl-form__label vl-u-spacer-extended-right-s vlc-u-text--nowrap" for="dossierId">
+          {{t "agendaitems-page-size"}}
+        </label>
+        <div class="vlc-vlc-search__sizeoptions">
+          {{#power-select
```

Comment:
Deze route is al volledig Octane, ook met `<AngleBrackets/>` in de template. Graag dat bij updates ook zo houden. Graag ook de [to-be deprecated `{{action ...}}` helper](https://guides.emberjs.com/release/upgrading/current-edition/action-on-and-fn/) niet meer gebruiken.   [](https://www.pzuraq.com/ember-octane-update-action/)

Reply:
fixed.